frontend/server/src/DAO/QualityNominations.php

```diff
@@ -190,6 +190,76 @@ private static function getVotesForNomination(int $qualitynomination_id) {
         return $votes;
     }
 
+    /**
+     * Returns the quality seal and id of a nomination for a given problem and user.
+     *
+     * @return array{quality_seal: bool, qualitynomination_id: int}
+     */
+    public static function getReviewedData(
+        int $problemId,
+        int $userId
+    ): array {
+        $sql = "
+            SELECT
+                qn.contents,
+                qn.qualitynomination_id
+            FROM
+                QualityNominations qn
+            INNER JOIN
+                Identities i ON i.user_id = qn.user_id
+            WHERE
+                nomination = 'quality_tag' AND
+                i.user_id = ? AND
+                qn.problem_id = ?;";
+        $params = [
+            $userId,
+            $problemId,
+        ];
+
+        /** @var array{contents: string, qualitynomination_id: int}|null */
+        $row = \OmegaUp\MySQLConnection::getInstance()->GetRow($sql, $params);
+
+        if (is_null($row)) {
+            return [
+                'quality_seal' => false,
+                'qualitynomination_id' => 0,
+            ];
+        }
+
+        /** @var array{quality_seal: bool} */
+        $contents = json_decode($row['contents'], true);
+        return [
+            'quality_seal' => boolval($contents['quality_seal']),
+            'qualitynomination_id' => intval($row['qualitynomination_id']),
```

Comment:
```suggestion
            'quality_seal' => $contents['quality_seal'],
            'qualitynomination_id' => $row['qualitynomination_id'],
```

estos tipos ya se declararon antes.